web/templates/alerts.html

```diff
@@ -9,19 +9,20 @@
     <table class="table table-striped table-bordered table-hover">
       <thead>
         <tr>
+          <th class="alert_expand_col"></th>
           <th>Alert</th>
           <th>Labels</th>
+          <th>Summary</th>
```

Comment:
Summary is user-provided and could end up very long, I'd show it only when exapnded.